src/main/java/nl/mpdev/backend_spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,78 @@
+package nl.mpdev.backend_spring_boot_tech_it_easy_controller.controllers;
+
+import nl.mpdev.backend_spring_boot_tech_it_easy_controller.exceptions.RecordNotFoundException;
+import nl.mpdev.backend_spring_boot_tech_it_easy_controller.exceptions.StringTooLongException;
+import nl.mpdev.backend_spring_boot_tech_it_easy_controller.models.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1")
+public class TelevisionsController {
+
+  List<Television> televisionDataBase = new ArrayList<>();
+
+  @GetMapping("/televisions")
+  public ResponseEntity<List<Television>> getAllTelevisions() {
+//    Map<String, String> response = new HashMap<>();
+//    response.put("television 1","Toshiba");
+//    response.put("television 2", "LG");
+    return ResponseEntity.ok(televisionDataBase);
+  }
+
+  @GetMapping("/televisions/{id}")
+  public ResponseEntity<Television> getTelevisionById(@PathVariable("id") int id) {
+    if (findTelevisionById(id) == null) {
+      throw new RecordNotFoundException();
+    }
+    return ResponseEntity.ok(findTelevisionById(id));
+  }
+
+  @PostMapping("/televisions")
+  public ResponseEntity<Integer> postTelevision(@RequestBody String television) {
+    if (television.length() > 20) {
+      throw new StringTooLongException("This string is way too long");
+    }
+    Television newTV = new Television(television);

```

Comment:
Je maakt hier een nieuw Television object aan de hand van de RequestBody. Dat is helemaal prima, maar je had misschien makkelijker een Television object in de RequestBody kunnen opvragen in plaats van een String